libs/application/ui-fields/src/lib/TableRepeaterFormField/TableRepeaterFormField.tsx

```diff
@@ -372,3 +284,190 @@ export const TableRepeaterFormField: FC<Props> = ({
     </Box>
   )
 }
+
+interface ItemFieldProps {
+  application: Application
+  error?: string
+  item: TableRepeaterItem & { id: string }
+  dataId: string
+  activeIndex: number
+  values: Array<Record<string, string>>
+}
+
+const Item: FC<ItemFieldProps> = ({
+  application,
+  error,
+  item,
+  dataId,
+  activeIndex,
+  values,
+}) => {
+  const { formatMessage } = useLocale()
```

Comment:
```suggestion
const Item = ({
  application,
  error,
  item,
  dataId,
  activeIndex,
  values,
}: ItemFieldProps) => {
  const { formatMessage } = useLocale()
```

Nitpick:

We should me moving away from using `FC` to define props, especially if the component renders no `children`, since FC implicitly adds the `children` type to props, meaning this would be legal code, even though no children are rendered:

```ts
<Item>hello world</Item>
```